tmp/lanelet2_extension/README.md

```diff
@@ -39,6 +39,10 @@ Autoware Traffic Light class contains following members:
 - light bulbs information of traffic lights
 - stopline associated to traffic light
 
+#### RightOfWay
```

Comment:
In a normal text, I prefer to keep it with space expression rather than CamelCase.
```suggestion
#### Right of Way
```